scripts/prepare-release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+PREVIOUS_VERSION="$(cat deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | grep integreatly-operator | awk -F v '{print $2}')"
+
+create_new_csv() {
+  operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name integreatly-operator --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
+}
+
+set_csv() {
+  if [[ -n "$VERSION" && -n "$TAG" ]]; then
+    sed -i "s/containerImage:.*/containerImage: quay\.io\/integreatly\/integreatly-operator:v$VERSION/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+    sed -i "s/image:.*/image: quay\.io\/integreatly\/integreatly-operator:v$VERSION-$TAG/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
+    sed -i "s/version: $VERSION.*/version: $VERSION-$TAG/g" deploy/olm-catalog/integreatly-operator/${VERSION}/integreatly-operator.v${VERSION}.clusterserviceversion.yaml
```

Comment:
What's this for? The version in the CSV should be set by create_new_csv no?

Reply:
It does, but I'm treating tag and version seperatly, otherwise the csv would be generated as `integreatly-operator.v9.9.9-<tag>.clusterserviceversion.yaml` which is incorrect

Reply:
Hmm, i see this has been set to include er1 alright, but i'm not sure why https://github.com/integr8ly/integreatly-operator/blob/master/deploy/olm-catalog/integreatly-operator/integreatly-operator-2.1.0/integreatly-operator.v2.1.0.clusterserviceversion.yaml#L404

@philbrookes @maleck13 Any reason we are setting this version to include a suffix for the build?

Reply:
I didn't do that for 2.0.0, I think that happened in 2.1.0 as a result of @wei-lee demo. I don't feel that this needs to be here.

Reply:
@StevenTobin We can drop this part of the update as it's not required, which will let us simplify this script as described here https://github.com/integr8ly/integreatly-operator/pull/629/files#r413598609